Packs/FiltersAndTransformers/Scripts/IsInCidrRanges/README.md

```diff
@@ -17,5 +17,5 @@ addresses can be passed as comma-delimited list to be checked.
 
 | **Argument Name** | **Description** |
 | --- | --- |
-| left | The IPv4 address to search for. |
-| right | A comma-separated list of IPv4 ranges in CIDR notation against which to match. |
+| left | The IPv4 or IPv6 address to search for. |
+| right | A comma-separated list of IPv4 or IPv6 ranges in CIDR notation against which to match. |
```

Comment:
This should be updated in the yml as well.

Reply:
[Done](https://github.com/demisto/content/pull/33246/commits/1b5d3705ab3cf317b72dfb3477d91d330e273d03).